test/build.py

```diff
@@ -39,66 +45,64 @@ def ensure_wasm_executable(path_to_wasm):
         sys.exit(1)
 
 
-def ensure_clean_initial_state():
-    ensure_wasm_executable(WASM_EXEC)
-
-    if os.path.exists(OUT_DIR):
-        shutil.rmtree(OUT_DIR)
-
-    ensure_dir(OUT_DIR)
-    ensure_dir(OUT_JS_DIR)
-    ensure_dir(OUT_HTML_DIR)
-
 # JS harness.
-def replace_js_harness(js_file):
-    test_func_name = os.path.basename(js_file).replace('.', '_').replace('-', '_')
+def convert_one_wast_file(inputs):
+    wast_file, js_file = inputs
+    print('Compiling {} to JS...'.format(wast_file))
+    return run(WASM_EXEC, wast_file, '-h', '-o', js_file)
```

Comment:
The output JS file is intended to be used in both a pure JS environment (shell) and the WPT environment; the WPT environment redefines all the testing primitives, as you know.